bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -15,27 +15,43 @@
 		<item-type>Number:Illuminance</item-type>
 		<label>Illuminance</label>
 		<description>Current illuminance.</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
 	<channel-type id="light_level" advanced="true">
 		<item-type>Number</item-type>
 		<label>Light Level</label>
 		<description>Current light level.</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
 		<state readOnly="true" pattern="%d"/>
 	</channel-type>
 
 	<channel-type id="dark">
 		<item-type>Switch</item-type>
 		<label>Dark</label>
 		<description>Light level is below the darkness threshold.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Light</tag>
```

Comment:
Property tags are supposed to be used with "Control" and "Measurement" points, but not with "Status" (I know, this isn't clearly documented anywhere, sorry...).

Reply:
Tags removed